Lib/ufo2ft/fontInfoData.py

```diff
@@ -11,11 +11,16 @@
 used externally as well.
 """
 
+from __future__ import print_function, division, absolute_import, unicode_literals
+from fontTools.misc.py23 import tounicode, unichr
```

Comment:
Importing unichr from fontTools.misc.py23 sometimes raises an error. Seems to have something to do with the Python installation, when overwriting the system unichr isn't necessary? But anyways, this should be handled (here or in FontTools).